docs/evm/how-it-works.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Differences How EVM on Flow Works
+sidebar_label: How it Works
+sidebar_position: 2
+---
+
+# Differences between EVM on Flow and Ethereum
+
+## Introduction
+
+The Flow network uses [Cadence](https://cadence-lang.org/) as its main execution environment. Cadence offers a safe, efficient, and developer-friendly experience for building smart contracts and decentralized applications. With Cadence offering full EVM support existing applications and tools already deployed for the EVM ecosystem can simply onboard to the Flow network with no code changes ([more](https://developers.flow.com/evm/about) on this topic).
```

Comment:
Typo, would update to:

The Flow network uses [Cadence](https://cadence-lang.org/) as its main execution environment. Cadence offers a safe, efficient, and developer-friendly experience for building smart contracts and decentralized applications. Cadence can be used to extend EVM apps built in solidity by unlocking gaslesss experiences, new business models and fine-tuned access control. With Flow offering full EVM support, existing applications and tools already deployed in the EVM ecosystem can simply onboard to the network with [no code changes](https://developers.flow.com/evm/about).

Reply:
updated